src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
So, what I've been trying to do with the new knobs is avoiding changing settings on systemd side underneath the users. MemoryMax and MemoryLimit are translated back and forth when they're applied to the kernel cgroup settings but the two knobs are still independent from each other and setting one value doesn't affect the other in any way. Otherwise, we end up with situations where setting one knob instantiates multiple knobs or loading order issues between old and new knobs if both are defined.

As for logging, I agree that adding debug messages for each translation and one time warning for the first translation would be a good idea. Can I do that in a separate PR tho? We need to update IO along anyway.